docs/command reference.md

```diff
@@ -133,3 +133,11 @@ psabpf-ctl register set pipe ID REGISTER_NAME index DATA value REGISTER_VALUE
 
 REGISTER_VALUE := { DATA }
 ```
+
+# Value set
+
+```shell
+psabpf-ctl value-set insert pipe ID VALUE_SET_NAME value DATA
```

Comment:
Any reason to change convention from `add` to `insert` ? We use `add` for insertion in other commands. I'd keep it consistent.